src/helper/fetch-all.function.ts

```diff
@@ -11,7 +11,7 @@ import { ScanRequest } from '../dynamo/request/scan/scan.request'
  * available. This can be used with scan and query requests.
  */
 
-export function fetchAll<T>(request: ScanRequest<T> | QueryRequest<T>, startKey?: DynamoDBv2.Key): Promise<T[]> {
+export function fetchAll<T>(request: ScanRequest<T> | QueryRequest<T>, startKey?: Record<string, AttributeValue>): Promise<T[]> {
```

Comment:
let's try to stick with DynamoDB.AttributeValue to be consistent (`import * as DynamoDB from '@aws-sdk/client-dynamodb'`)

Reply:
I also updated the `KeyType` imports to be consistent.